var/spack/repos/builtin/packages/pfft/package.py

```diff
@@ -27,46 +27,50 @@ def configure(self, spec, prefix):
 
         configure = Executable('../configure')
 
-        if '+double' in spec['fftw']:
+        fftw_precisions = spec['fftw'].variants.get('precision')
```

Comment:
I've never seen this syntax before, does this work? I've only ever seen `spec['fftw'].variants['precision'].value`